scss/_variables.scss

```diff
@@ -626,27 +626,25 @@ $tooltip-arrow-color:               $tooltip-bg !default;
 
 // Popovers
 
-$popover-inner-padding:             1px !default;
 $popover-bg:                        $white !default;
-$popover-max-width:                 276px !default;
+$popover-max-width:                 17.5rem !default;
 $popover-border-width:              $border-width !default;
 $popover-border-color:              rgba($black,.2) !default;
-$popover-box-shadow:                0 5px 10px rgba($black,.2) !default;
+$popover-box-shadow:                0 .25rem .5rem rgba($black,.2) !default;
 
 $popover-header-bg:                 darken($popover-bg, 3%) !default;
 $popover-header-color:              $headings-color !default;
-$popover-header-padding-y:          8px !default;
-$popover-header-padding-x:          14px !default;
+$popover-header-padding-y:          .5rem !default;
+$popover-header-padding-x:          .75rem !default;
 
 $popover-body-color:                $body-color !default;
-$popover-body-padding-y:            9px !default;
-$popover-body-padding-x:            14px !default;
+$popover-body-padding-y:            $popover-header-padding-y !default;
+$popover-body-padding-x:            $popover-header-padding-x !default;
 
-$popover-arrow-width:               10px !default;
-$popover-arrow-height:              5px !default;
+$popover-arrow-width:               .8rem !default;
```

Comment:
Same for these two?

Reply:
@XhmikosR I would say for the max-width since we measure grids in `px` but not for the size of the arrow. But I am not sure, What do you think?

Reply:
@XhmikosR actually, if the root font size changes, we want to space of the  popover to be larger.
So, maybe the max-width should in rems too, right?